tests/Faithlife.Utility.Tests/OptionalTests.cs

```diff
@@ -48,7 +48,9 @@ public void TestEqualsOptionalNullAndNull()
 		[Test]
 		public void TestEqualsNoneAndNull()
 		{
+#pragma warning disable CS8625 // Cannot convert null literal to non-nullable reference type.
```

Comment:
`null!` should also work and is preferable.

Reply:
Much better! Thanks!